module_report/check_modules.py

```diff
@@ -0,0 +1,46 @@
+from pathlib import Path
+import pkgutil
+
+import qtpy
+
+
+QTPY_MODULES = [module.name for module in pkgutil.iter_modules(qtpy.__path__)
+                if module.name.startswith(Qt)]
+
+
+QT_MODULES = {}
+
+FALSE_POSITIVES = [
+    'QtChart',
+    'QtOpenGLFunctions',
+    'QtScript',
+    'QtScriptTools',
+]
+
+
+for api_name in qtpy.API_NAMES.values():
+    try:
+        api_path = pkgutil.resolve_name(api_name).__path__
+    except ImportError:
+        continue
+    api_modules = [mod.name for mod in pkgutil.iter_modules(api_path)]
+    api_modules =  [mod for mod in api_modules if mod.name.startswith('Qt')]
```

Comment:
```suggestion
    api_modules =  [mod for mod in api_modules if mod.startswith('Qt')]
```

There's probably more, but I noticed I made a mistake here when refactoring this

Reply:
Also already fixed locally.